### PR TITLE
[KYUUBI #1067]catch exception if logRoot is not exist

### PR DESCRIPTION
**Pull request by zhang1002**
BUGFIX:
#1067 
This PR is just for version_1.3.0

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by yaooqinn — APPROVED** [review]

File: externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala [hunk 1]
```diff
@@ -65,8 +65,10 @@ case class SparkSQLEngine(spark: SparkSession) extends Serverable("SparkSQLEngin
   }
 
   override def stop(): Unit = {
-    eventLogging.onEvent(
-      engineStatus.copy(state = ServiceState.STOPPED.id, endTime = System.currentTimeMillis()))
+    if (!state.equals(ServiceState.LATENT)) {
```
Comment (ulysses-you) [inline externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala:68]: should it be `if (state == ServiceState.STARTED)` ?

Comment (zhang1002) [reply externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala:68]: i think state judgement should in the method of JsonEventLogger.getOrUpdate.
only "STARTED" state can provided this service.

this need @yaooqinn to confirm.
